### PR TITLE
social media ticker v1

### PR DESCRIPTION
**Pull request by ric96**
96Boards Social Ticker project Version 1

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/)
+- [SeeedStudio Sensors Mezzanine](https://www.96boards.org/product/sensors-mezzanine/)
+- 5x [MAX7219 LED Matrix Modules](https://www.amazon.com/MAX7219-Microcontroller-Compatible-Atomic-Market/dp/B00TNNDH0A/ref=sr_1_13?ie=UTF8&qid=1541425999&sr=8-13&keywords=max7219)
+- [SSD1306 OLED 128x64](https://www.adafruit.com/product/326)
+## 1.2) Hardware Setup
+- Connect all the LED matrix Modules in a daisy-chain configuration
+- Connect the Sensor Mezzanine to the first LED Matrix Module as follow:
+| Sensor Mezzanine | MAX7219 LED Matrix |
```
Comment (Mani-Sadhasivam) [inline README.md:25]: table rendering here and below is not proper!

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+
+- Connect the OLED Module as follow:
+| Sensor Mezzanine | SSD1306 OLED |
+|:----------------:|:------------:|
+| 5v               | VCC          |
+| GND              | GND          |
+| SDA              | SDA          |
+| SCL              | SCL          |
+
+- I use ISCP to program the SeeedStudio Mezzanine. You can follow [this guide](https://www.96boards.org/blog/arduino-ide-sensor-mezzanine/).
```
Comment (Mani-Sadhasivam) [inline README.md]: s/use ISCP/used ICSP programmer

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Please add description and _always_ sign off. This holds true for both PR and commits.

**Comment by ric96** [comment]
@Mani-Sadhasivam can you take a look? I've updated as required.

**Review by Mani-Sadhasivam — APPROVED** [review]

**Pull request merged**